Added more information on Okta hosted and Customer hosted concepts (#2324)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md

## Okta Hosted Vs. Customer Hosted

> Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. You can customize the Okta URL domain and the Okta-hosted or Customer-hosted Sign-In Widget style. See [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/) and [Style the Widget](/docs/guides/style-the-widget/before-you-begin/).

Reviewer: ```suggestion
Okta offers multiple integration options for designing an authentication experience. The two most recommended options are integrating the Okta-hosted Sign-In Widget and the customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. You can customize the Okta URL domain and the Okta-hosted or customer-hosted Sign-In Widget style. See [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/) and [Style the Widget](/docs/guides/style-the-widget/before-you-begin/).
```

Reviewer: Not sure if "Customer-hosted Sign-In Widget" is a product? I am assuming that "Sign-In Widget" is the product and that customer-hosted is the modifier. Change this if my assumptions are incorrect. Thanks!

> The difference between the two concepts is where the Sign-In Widget itself is hosted. See [Okta-hosted widget](/docs/concepts/hosted-vs-embedded/#okta-hosted-widget) and [Customer-hosted widget](/docs/concepts/hosted-vs-embedded/#customer-hosted-widget) for diagrams on where the Sign-In Widget is hosted.

Reviewer: ```suggestion
The difference between the two integration options is where the Sign-In Widget is hosted. See [Okta-hosted widget](#okta-hosted-widget) and [Customer-hosted widget](#customer-hosted-widget) for hosted scenarios.
```

Reviewer: I didn't see diagrams in the topic. If the diagrams are to be added, then please revert this suggestion. Thanks

Author: The diagrams are in the sections as referenced.

> Another difference is when you're using the Okta-hosted Sign-In widget, you're communicating with Okta using OAuth and OIDC. When you're using the Customer-hosted Sign-In widget, you're using the Javascript SDK and communication with Okta is handled for you transparently.

Reviewer: ```suggestion
Another difference between the two options is the method of communication. When you're using the Okta-hosted Sign-In Widget, you're communicating with Okta using OAuth 2.0 and OIDC. When you're using the customer-hosted Sign-In Widget, you're using the Okta JavaScript SDK and communication with Okta is handled for you transparently.
```

## Configure policies for app sign-in experiences

> For both Okta-hosted and Customer-hosted sign-in experiences, you can configure the policies in the Okta Admin console to determine how the app will behave. 

Reviewer: ```suggestion
For both Okta-hosted and customer-hosted sign-in experiences, you can configure policies in the Okta Admin Console to determine the behavior of the app. 
```

1. Go to **Security** > **API**. 
1. In the **Authorization Servers** tab, select an authorization server and click the edit icon beside it. 

> 1. On the authorization server page, select the **Access Policies** tab and scroll down to the Rules

Reviewer: ```suggestion
1. On the authorization server page, select the **Access Policies** tab and scroll down to the Rules list.
```

> 1. Select a Rule, and click the edit icon beside it. The Edit Rule window appears where you can update the settings for the access policies.

Reviewer: ```suggestion
1. Click the edit icon beside the Rule you want to modify. The Edit Rule window appears where you can update the settings for the access policies.
```
